templates/cluster/main.yml.erb

```
@@ -242,13 +242,23 @@ v2:
 <% end -%>
 <% if ! @batch_connect.empty? -%>
   batch_connect:
-  <%- if @batch_connect.key?('basic') -%>
+  <%- if @batch_connect.key?('basic') and @batch_connect['basic'].size > 0 -%>
     basic:
-      script_wrapper: "<%= @batch_connect['basic']['script_wrapper'] %>"
+    <%- if @batch_connect['vnc'].key?('script_wrapper') -%>
```

Should the key be `basic`?  Suggestion in case this is a typo
```suggestion
    <%- if @batch_connect['basic'].key?('script_wrapper') -%>
```